Cargo.toml

```diff
@@ -11,5 +11,6 @@ repository = "https://github.com/Sensirion/lin-bus-driver-serial-rs"
 edition = "2018"
 
 [dependencies]
-lin-bus = "^0.2"
+#lin-bus = "^0.3"
+lin-bus = { git = "https://github.com/sensirion/lin-bus-rs.git", branch="prepare-release" }
```

Comment:
Change after https://github.com/Sensirion/lin-bus-rs/pull/22 is merged, tagged and published to crates.io